ray-operator/config/samples/ray_v1alpha1_rayservice.yaml

```diff
@@ -3,4 +3,94 @@ kind: RayService
 metadata:
   name: rayservice-sample
 spec:
-  # TODO(user): Add fields here
+  serveConfigs:
+    - name: shallow
+      import_path: "test_env.shallow_import.ShallowClass"
+      num_replicas: 1
+      route_prefix: "/shallow"
+      ray_actor_options:
+        num_cpus: 0.1
+        runtime_env:
+          py_modules:
+            - "https://github.com/shrekris-anyscale/test_deploy_group/archive/HEAD.zip"
+            - "https://github.com/shrekris-anyscale/test_module/archive/HEAD.zip"
+    - name: deep
+      import_path: "test_env.subdir1.subdir2.deep_import.DeepClass"
+      num_replicas: 1
+      route_prefix: "/deep"
+      ray_actor_options:
+        num_cpus: 0.1
+        runtime_env:
+          py_modules:
+            - "https://github.com/shrekris-anyscale/test_deploy_group/archive/HEAD.zip"
+            - "https://github.com/shrekris-anyscale/test_module/archive/HEAD.zip"
+    - name: one
+      import_path: "test_module.test.one"
+      num_replicas: 1
+      ray_actor_options:
+        num_cpus: 0.1
+        runtime_env:
+          py_modules:
+            - "https://github.com/shrekris-anyscale/test_deploy_group/archive/HEAD.zip"
+            - "https://github.com/shrekris-anyscale/test_module/archive/HEAD.zip"
+  rayClusterConfig:
+    rayVersion: '1.8.0' # should match the Ray version in the image of the containers
+    ######################headGroupSpecs#################################
+    # head group template and specs, (perhaps 'group' is not needed in the name)
+    headGroupSpec:
+      # Kubernetes Service Type, valid values are 'ClusterIP', 'NodePort' and 'LoadBalancer'
+      serviceType: ClusterIP
+      # the pod replicas in this group typed head (assuming there could be more than 1 in the future)
+      replicas: 4
+      # logical group name, for this called head-group, also can be functional
+      # pod type head or worker
+      # rayNodeType: head # Not needed since it is under the headgroup
+      # the following params are used to complete the ray start: ray start --head --block --redis-port=6379 ...
+      rayStartParams:
+        port: '6379' # should match headService targetPort
+        #include_webui: 'true'
+        object-store-memory: '100000000'
+        redis-password: 'LetMeInRay' # Deprecated since Ray 1.11 due to GCS bootstrapping enabled
```

Comment:
After updating the Ray version in this example config, we can drop this argument.